src/storage/patch/SparqlUpdatePatchHandler.ts

```diff
@@ -35,15 +35,15 @@ export class SparqlUpdatePatchHandler extends PatchHandler {
 
   public async canHandle(input: {identifier: ResourceIdentifier; patch: SparqlUpdatePatch}): Promise<void> {
     if (typeof input.patch.algebra !== 'object') {
-      throw new UnsupportedHttpError('Only SPARQL update patch requests are supported');
+      throw new BadRequestHttpError('Only SPARQL update patch requests are supported');
```

Comment:
This is an example of one where I'm not sure (it pretty much comes down to: does the spec say only those are supported or have we simple not added functionality for the other ones yet? Related to #15).

Reply:
Was a clear `NotImplementedError` for me.